clients/authorization-api-v1/README.md

```diff
@@ -15,13 +15,13 @@ npm install @sp-api-sdk/authorization-api-v1
 ## Getting Started
 
 ```javascript
-import {SellingPartnerApiAuth} from '@sp-api-sdk/auth'
+import {SellingPartnerApiAuth, AuthorizationScopeLWA} from '@sp-api-sdk/auth'
 import {AuthorizationApiClient} from '@sp-api-sdk/authorization-api-v1'
 
 const auth = new SellingPartnerApiAuth({
   clientId: '',
   clientSecret: '',
-  refreshToken: '',
+  scopes: Object.values(AuthorizationScopeLWA), // Or choose the only ones you need
```

Comment:
I think we should actually show the real scope here. => AuthorizationScope.MIGRATION